bin/start

```diff
@@ -2,11 +2,15 @@
 
 main(){
   case $1 in
-    -i) #start normally
+    -i) #start normally - used for jenkins
       docker-compose up -d
+      docker exec $(docker-compose ps -q gridium) bundle install;
```

Comment:
except there's not.  In order to auto-increment the patch version from the build number, we can no longer bundle install from the Dockerfile.